src/suites/cts/validation/error_scope.spec.ts

```diff
@@ -94,11 +112,15 @@ g.test('push/popping error scopes must be balanced', async t => {
     const promise = t.device.popErrorScope();
     await t.shouldReject('OperationError', promise);
   }
-  t.device.pushErrorScope('validation');
-  {
-    const error = await t.device.popErrorScope();
-    t.expect(error === null);
+
+  const promises = [];
+  for (let i = 0; i < 1000; i++) {
+    t.device.pushErrorScope('validation');
+    promises.push(t.device.popErrorScope());
```

Comment:
This does 1000 sibling error scopes, can we add another test for 1000 nested error scopes?

Reply:
Sure.